Add link with lapack (via jrl-mal.pc), for binutils>=2.22

For some reason, the Libs: from jrl-mal.pc end up in LDFLAGS_OTHER, not
LIBRARIES.

--- src/CMakeLists.txt~	2011-12-07 15:35:56.000000000 +0100
+++ src/CMakeLists.txt	2012-02-13 14:43:59.762476633 +0100
@@ -57,6 +57,7 @@
 ENDIF (UNIX)
 
 TARGET_LINK_LIBRARIES(${LIBRARY_NAME} ${Boost_LIBRARIES})
+TARGET_LINK_LIBRARIES(${LIBRARY_NAME} ${JRL_MAL_LDFLAGS_OTHER})
 
 INSTALL(TARGETS ${LIBRARY_NAME}
   DESTINATION lib)
